### ISSUE TITLE
Locale  Language File

### ISSUE THREAD

**Issue by sonofachilles9** [tastybento/ASkyBlock-Bugs-N-Features#350]
I have this set in the ASkyBlock Config
   defaultlanguage: en-US

and i have edited the entire en-US to change Island to Empire i have reloaded and restarted the server but when i do /is restart i get "Type /island confirm instead of empire. Plus also the text when you first make an island still appears in middle of the screen even though i have already removed it. How come this is?

**Comment by sonofachilles9** [comment]
Also can i disable /is help? or change the list of /is help because i disabled commands from being used yet players can still do /is help and see massive list of commands. so i was hoping maybe we could take out commands they cant use so it shows only the commands they can

**Comment by Poslovitch** [comment]
You cannot edit the commands (/island, /asc, /asadmin). You'll need to fork the code, add an alias in plugin.yml and build your version.

Disabling commands will be possible in bSkyblock.

**Comment by sonofachilles9** [comment]
Any idea of a realse date of bskyblock?

**Comment by Poslovitch** [comment]
Join our Discord server ( https://discord.gg/ur8EPYS). Almost everything is said on it.

**Comment by tastybento** [comment]
@Poslovitch is right, you can't change the fundamental commands. But you should be able to change that message okay. I think you're player is probably not viewing the US English file. The default language set in config.yml is what new players will see. If you want to see it, make sure you select it for your player by doing /island lang and then the number (I think it's #4). Also, it may be that you messed up the YML in the language file so it is using the default settings. Check your file with a YAML syntax checker (google it).

**Comment by sonofachilles9** [comment]
I have checked the YML and all is fine. i get a green tick to say the yml is valid here is a screenshot of the issue i want to correct

https://gyazo.com/f24e52b5a6240d96b550291798e98439

I want to fully remove that big sign on island create. and also in chat the /island confirm i want as /empire confirm. Here is the YML for the EN-US which my account IS using as i double checked with /is lang 10 which is English USA

Here is my entire en.USA lang file YML for you to see what i have got and why it might not be changing ingame 

https://pastebin.com/XkfySBGZ

**Comment by tastybento** [comment]
There are a number of issues with what you are trying to do:

1. You cannot change the commands of the game using the locale file. So, for example, the game commands will always be **/island** or **/island deleteisland**, etc. To change the commands, you will need to edit the source code of the plugin and the plugin.yml file. 
2. You cannot change the keys of the yml file, that is, the words before the colons. If these keys are missing, the plugin will use the default string in the plugin. So, for example you have this:

**adminDeleteEmpire:** but it must remain **adminDeleteIsland:** 

In other words, you can change the text inside the quotes, but nothing else. It looks like you did a global search and replace of Island for Empire and so you need to go through it more carefully. 

Also, watch out, keys are case sensitive! So, on line 270, you have Island:, but it must be island: this is why the welcome text is not changing, that whole section is being ignored.

3. To change the welcome text of the island, you correctly changed title to "", but also search for subtitle: "by tastybento" and change that to subtitle: "" as well.

**Comment by sonofachilles9** [comment]
That's fair enough but we have a theme for our server which is medieval so we want to mask out completely "Island" which is why I'm trying to change everything to empire. Doesn't really suit our theme when we have managed to mask majority out to empire but then some still appear as "island" when it comes to resetting the island. I only want the text that it says there to say "Empire" nothing more. That shouldn't change the command as the command still works and we just aliased it to Empire. Can we not even change that text to Empire? And is there anyway you could possibly send us a Lang file which does this for us? Or maybe a reset Lang file so I can re do it?

Appreciate it mate :)

Sent from my iPhone

On 20 Jul 2017, at 21:19, tastybento <notifications@github.com<mailto:notifications@github.com>> wrote:


There are a number of issues with what you are trying to do:

  1.  You cannot change the commands of the game using the locale file. So, for example, the game commands will always be /island or /island deleteisland, etc. To change the commands, you will need to edit the source code of the plugin and the plugin.yml file.
  2.  You cannot change the keys of the yml file, that is, the words before the colons. If these keys are missing, the plugin will use the default string in the plugin. So, for example you have this:

adminDeleteEmpire: but it must remain adminDeleteIsland:

In other words, you can change the text inside the quotes, but nothing else. It looks like you did a global search and replace of Island for Empire and so you need to go through it more carefully.

Also, watch out, keys are case sensitive! So, on line 270, you have Island:, but it must be island: this is why the welcome text is not changing, that whole section is being ignored.

  1.  To change the welcome text of the island, you correctly changed title to "", but also search for subtitle: "by tastybento" and change that to subtitle: "" as well.

**Comment by sonofachilles9** [comment]
Or is it possibly I can edit the source code and change the commands to Empire so I don't need to alias it? Is it easy done? And is it something you may allow us to do?

Sent from my iPhone

On 20 Jul 2017, at 21:19, tastybento <notifications@github.com<mailto:notifications@github.com>> wrote:


There are a number of issues with what you are trying to do:

  1.  You cannot change the commands of the game using the locale file. So, for example, the game commands will always be /island or /island deleteisland, etc. To change the commands, you will need to edit the source code of the plugin and the plugin.yml file.
  2.  You cannot change the keys of the yml file, that is, the words before the colons. If these keys are missing, the plugin will use the default string in the plugin. So, for example you have this:

adminDeleteEmpire: but it must remain adminDeleteIsland:

In other words, you can change the text inside the quotes, but nothing else. It looks like you did a global search and replace of Island for Empire and so you need to go through it more carefully.

Also, watch out, keys are case sensitive! So, on line 270, you have Island:, but it must be island: this is why the welcome text is not changing, that whole section is being ignored.

  1.  To change the welcome text of the island, you correctly changed title to "", but also search for subtitle: "by tastybento" and change that to subtitle: "" as well.

**Comment by tastybento** [comment]
If you've already aliased the command to /empire, then you should be good. Just adjust the locale file carefully as I mentioned, i.e., you cannot change the keys, just the values. For the admin commands, just leave them as island as your players won't see them.

**Comment by sonofachilles9** [comment]
Thanks. Can you poss send me a fresh en-US locale file please matey :)

Sent from my iPhone

On 21 Jul 2017, at 02:14, tastybento <notifications@github.com<mailto:notifications@github.com>> wrote:


If you've already aliased the command to /empire, then you should be good. Just adjust the locale file carefully as I mentioned, i.e., you cannot change the keys, just the values. For the admin commands, just leave them as island as your players won't see them.

**Comment by tastybento** [comment]
It's on Github here (just search for Askyblock) but also if you move or delete the locale folder and restart or reload all the files will be regenerated fresh.
